smartcontract/service/native/ontid/owner.go

```diff
@@ -91,7 +93,7 @@ func insertPk(srvc *native.NativeService, encID, pk []byte) (uint32, error) {
 		owners = make([]*owner, 0)
 	}
 	size := len(owners)
-	if size >= 0xFFFFFFFF {
+	if size >= OWNER_MAX_SIZE {
```

Comment:
add `pk` size check in all places.

Reply:
pk is validated in the interfaces.
It is not flexible to make a fixed size limit here.

Reply:
then check the total byte size of the key list instead of the amount.